business/statusService.js

```diff
@@ -109,6 +110,29 @@ class StatusService {
     })
   }
 
+  async _crawlbreakdown() {
+    const data = await requestPromise(
+      this._crawlerQuery(`
+      traces      
+      | where customDimensions.outcome == 'Processed'  
```

Comment:
can you add a line to filter where crawler host is defined?
`| where strlen(customDimensions.crawlerHost) > 0`

it's a bit of a hack, but we get a lot of double log statements that will throw off the counts without this

Reply:
Had noticed it on the other queries, was unsure it’d apply. Understood.